Formatting and some Day 2 updates (#77)
cu-psych-computing/cu-psych-comp-tutorial: content/tutorials/r-core/2-datacleaning/index.rmd

Sometimes, data might be spread across multiple spreadsheets, and you'll want to combine those for your analysis. For example, maybe this study had a follow-up survey on Day 30. Scores from that survey were entered into another spreadsheet, which only has the subject ID and that score. We want to include that score into our data.

Although there are many different functions you can use to combine data frames, here I'm using the function `inner_join()`, which is also part of the `tidyverse`.

```{r}
# First load the new data
mydata2 <- read_csv(here("content", "tutorials", "r-core", "2-dataCleaning", "Study1_Followup.csv"))

# Merge the two data frames. To make sure the rows match up, we use the 'by' argument to specify that IDs should match. That way even if the data is in a different order you will get scores matched togther correctly.

mydata <- inner_join(mydata, mydata2, by = "ID")

> # Note that mydata has one less row than mydata2 because we removed that outlier. By default, merge will use only cases of the 'by' variable that exist in both dataframes, so it drops that one row from mydata2 when merging. You can use additional arguments to change this if you want to keep all observations from one or both frames, even if it means blank cells.

Reviewer: Small update to this informational comment to reflect that we’re now teaching inner_join

```suggestion
# Note that mydata has one less row than mydata2 because we removed that outlier. By default, inner_join will use only cases of the 'by' variable that exist in both dataframes, so it drops that one row from mydata2 when merging. You can use other _join functions if you want to keep all observations from one or both frames, even if it means blank cells.
```
